docs/tutorials/intro_to_chemistry.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Intro To Chemistry Tutorial.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "N4327P0aWzVh",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Introduction to Electronic Structure Theory\n",
```

Comment:
This is a pretty heavy bit of math to put at the start of a tutorial. It's all correct and well-written, but I wonder if it will help someone who doesn't already know most of it. Could we perhaps shift some of this further down the tutorial to when it is first seen?

Reply:
Moved the math heavy section to an appendix at the bottom and added a smaller introductory section at the top. Let me know if it looks better now.